doc-botonic/docs/plugins/21.botonic-plugins.md

```diff
@@ -0,0 +1,138 @@
+---
+id: botonic-plugins
+title: Create a Botonic Plugin
+---
+
```

Comment:
Creating a botonic plugin should be considered for advanced developers. I don't think this should be the 1st thing in the plugins section.
Proposed index:
1) What are Botonic plugins and why are they useful (we can keep this short, take inspiration from https://www.gatsbyjs.org/docs/plugins)
2) Botonic Plugin Library (a list of all the plugins and how they work, much like https://www.gatsbyjs.org/plugins/)
3) Authoring Plugins (how to create your own plugin)

Reply:
To do in next sprint then.